CareKit/SymptomTracker/OCKSymptomTrackerViewController.h

```diff
@@ -166,6 +166,14 @@ OCK_CLASS_AVAILABLE
  */
 @property (nonatomic) NSString *customGlyphImageName;
 
+/**
+ A message that will be displayed in the table view's background view if there are
+ no intervention activities to display.
```

Comment:
Optional: A message that will be displayed in the table view's background view if there are no assessment activities to display.